lib/Agrammon/Web/Service.pm6

```diff
@@ -163,6 +163,23 @@ class Agrammon::Web::Service {
         my $with-filters = $type eq 'reportDetailed';
         my $all-filters  = $type eq 'reportDetailed';
 
+        my %submission;
+        if %params<mode> eq 'submission' {
+            my $sender-name = %params<senderName>;
+            # frontend converts newlines to XXX
+            $sender-name ~~ s:g/XXX/\\newline\{\}/;
```

Comment:
Hm, that feels a little odd...why?

Reply:
And if there can be more than one newline, needs to be `s:g`

Reply:
Well, it is `s:g`... I was wondering too, why I did this in the backend. I guess I can as well replace the `\n` directly in the PDF generator.

Reply:
Fixed (and removed the replacment from frontend).